app/ts/components/Settings.tsx

```diff
@@ -70,62 +85,94 @@ export const SettingsModal = ({ display, appSettings }: { display: Signal<boolea
+			submissionRelayEndpointInput.value = { value: bouquetNetwork.peek().submissionRelayEndpoint, valid: true }
+			priorityFeeInput.value = { value: formatUnits(bouquetNetwork.peek().priorityFee, 'gwei'), valid: true }
+			blocksInFutureInput.value = { value: bouquetNetwork.peek().blocksInFuture.toString(10), valid: true }
+			mempoolSubmitRpcEndpoint.value = { value: bouquetNetwork.peek().mempoolSubmitRpcEndpoint, valid: true }
+			relayMode.value = { value: bouquetNetwork.peek().relayMode, valid: true }
 		})
 	}
+
+	function resetSettings() {
+		localStorage.setItem('bouquetSettings', JSON.stringify(BouquetSettings.serialize(DEFAULT_NETWORKS)))
+		bouquetSettings.value = fetchSettingsFromStorage()
+		bringSettingsValues()
+	}
 	function close() {
-		batch(() => {
-			simulationRelayEndpointInput.value = { value: appSettings.peek().simulationRelayEndpoint, valid: true }
-			submissionRelayEndpointInput.value = { value: appSettings.peek().submissionRelayEndpoint, valid: true }
-			priorityFeeInput.value = { value: formatUnits(appSettings.peek().priorityFee, 'gwei'), valid: true }
-			blocksInFutureInput.value = { value: appSettings.peek().blocksInFuture.toString(10), valid: true }
-			display.value = false
-		})
+		display.value = false
 	}
-	return display.value ? (
+	return display.value && loaded.value ? (
 		<div onClick={close} className='bg-white/10 w-full h-full inset-0 fixed p-4 flex flex-col items-center md:pt-24'>
 			<div class='h-max px-8 py-4 w-full max-w-xl flex flex-col gap-4 bg-black' onClick={(e) => e.stopPropagation()}>
 				<h2 className='text-xl font-semibold'>App Settings</h2>
-				<div className={`flex flex-col justify-center border h-16 outline-none px-4 focus-within:bg-white/5 bg-transparent ${!simulationRelayEndpointInput.value.valid ? 'border-red-400' : 'border-white/50 focus-within:border-white/80'}`}>
-					<span className='text-sm text-gray-500'>Bundle Simulation Relay URL</span>
-					<input onInput={(e: JSX.TargetedEvent<HTMLInputElement>) => validateSimulationRelayEndpointInput(e.currentTarget.value)} value={simulationRelayEndpointInput.value.value} type='text' className='bg-transparent outline-none placeholder:text-gray-600' placeholder='https://' />
-				</div>
-				<div className={`flex flex-col justify-center border h-16 outline-none px-4 focus-within:bg-white/5 bg-transparent ${!submissionRelayEndpointInput.value.valid ? 'border-red-400' : 'border-white/50 focus-within:border-white/80'}`}>
-					<span className='text-sm text-gray-500'>Bundle Submission Relay URL</span>
-					<input onInput={(e: JSX.TargetedEvent<HTMLInputElement>) => validateAndSetSubmissionRelayEndpointInput(e.currentTarget.value)} value={submissionRelayEndpointInput.value.value} type='text' className='bg-transparent outline-none placeholder:text-gray-600' placeholder='https://' />
-				</div>
+				<label class = 'toggleSwitch nolabel' >
```

Comment:
There isn't a `nolabel` class in the stylesheet, removing
```suggestion
				<label class = 'toggleSwitch' >
```